imagepipeline/src/main/java/com/facebook/imagepipeline/producers/LocalVideoThumbnailProducer.java

```diff
@@ -113,4 +118,22 @@ private static int calculateKind(ImageRequest imageRequest) {
     }
     return MediaStore.Images.Thumbnails.MICRO_KIND;
   }
+
+  private String getLocalFilePath(ImageRequest imageRequest) {
+    Uri uri = imageRequest.getSourceUri();
+    if (UriUtil.isLocalFileUri(uri)) {
+      return imageRequest.getSourceFile().getPath();
+    } else if (UriUtil.isLocalContentUri(uri)) {
+      Cursor cursor = mContentResolver.query(uri, new String[] {MediaStore.Video.Media.DATA},
+        null, null, null);
+      try {
+        if (cursor.moveToFirst()) {
+          return cursor.getString(cursor.getColumnIndexOrThrow(MediaStore.Video.Media.DATA));
+        }
+      } finally {
+        cursor.close();
+      }
+    }
+    return "";
```

Comment:
Return `null` instead pls, and make sure we handle it correctly down the road (add `@Nullable` etc.)